pywinauto/controls/common_controls.py

```diff
@@ -3388,6 +3388,95 @@ def set_view(self, viewType):
         if res == 0:
             raise RuntimeError('Failed to set view in Calendar')
 
+    # ----------------------------------------------------------------
+    def set_id(self, ID):
+        """
+        Set the calendar type
+        Receive only one parameter, which takes variants below:
+        'gregorian', 'gregorian_US', 'japan', 'taiwan', 'korea',
+        'hijri', 'thai', 'hebwer', 'gregorian_french_me',
+        'gregorian_arabic', 'gregorian_english_xlit',
+        'gregorian_french_xlit', 'umalqura'
+        """
+
+        dict_types = {
+            'gregorian': 1,
+            'gregorian_US': 2,
+            'japan': 3,
+            'taiwan': 4,
+            'korea': 5,
+            'hijri': 6,
+            'thai': 7,
+            'hebwer': 8,
+            'gregorian_french_me': 9,
+            'gregorian_arabic': 10,
+            'gregorian_english_xlit': 11,
+            'gregorian_french_xlit': 12,
+            'umalqura': 23
+        }
+        if ID in dict_types:

```

Comment:
`ID` is not a built-in name in both Pythons. Lower case `id` is a built-in function.